singlem/singlem.py

```diff
@@ -58,6 +60,18 @@ def __init__(self, package_paths=None):
         for pkg in self.singlem_packages:
             self._hmms_and_positions[pkg.base_directory()] = pkg
 
+    def get_dmnd(self):
+        ''' Create temporary DIAMOND file for search method '''
+        fasta_paths = [pkg.graftm_package().unaligned_sequence_database_path() for pkg in self.singlem_packages]
+        temp_dmnd = tempfile.NamedTemporaryFile(mode="w", prefix='SM', 
```

Comment:
probably better to use `prefix='singlem-diamond-prefliter'` so it is very clear.

Do these tempfiles remain after singlem has finished? I suspect so.

Reply:
The tempfiles are created in the working directory, so they get cleaned up at the end along with everything else. They do remain if the program is stopped prematurely, however.

Reply:
true, ok, nvm then